app/src/main/java/me/hufman/androidautoidrive/carapp/carinfo/CarDetailedInfo.kt

```diff
@@ -74,27 +85,23 @@ class CarDetailedInfo(carCapabilities: Map<String, Any?>, cdsMetrics: CDSMetrics
 	val accelContact = cdsMetrics.accelerator.format("% 3d%%").map { "$it ${L.CARINFO_ACCEL}"}
 //	val clutchContact = cdsMetrics.clutch.map { "Clutch $it"}
 //	val brakeContact = cdsMetrics.brake.map { "Brake $it ${Integer.toBinaryString(it).padStart(8, '0')}"}
-	val steeringAngle = cdsMetrics.steeringAngle.map {
-		val icon = if (it <= -0.5) {"→"} else if (it >= 0.5) {"←"} else {"↔"}
-		"$icon % 3.0f°".format(it.absoluteValue)
-	}.map { "$it ${L.CARINFO_STEERING}" }
+	private val steeringAngle = cdsMetrics.steeringAngle.map {
+		(if (it <= -0.5) {"→"} else if (it >= 0.5) {"←"} else {"↔"}) +
+			" % 3.0f° ".format(it.absoluteValue) + L.CARINFO_STEERING
+	}
 	val speed = cdsMetrics.speedActual.format("% 3.0f ").addPlainUnit(unitsSpeedLabel)
 	val speedGPS = cdsMetrics.speedGPS.format("% 3.0f ").addPlainUnit(unitsSpeedLabel).map { "$it ${L.CARINFO_SPEEDGPS}"}
 	val torque =cdsMetrics.torque.format("% 3.0f Nm").map { "$it ${L.CARINFO_TORQUE}" }
 	val engineRpm = cdsMetrics.engineRpm.map { "$it ${L.CARINFO_RPM}"}
-	val heading = cdsMetrics.heading.map { heading ->
-		val direction = CDSMetrics.compassDirection(heading)
-		val arrow = CDSMetrics.compassArrow(heading)
-		"$arrow $direction (${heading.toInt()}°)"
-	}
-	private val gforceLat = cdsMetrics.accel.map { accel ->
-		accel.first?.let {
+	private val heading = cdsMetrics.heading.map { "${CDSMetrics.compassArrow(it)} ${CDSMetrics.compassDirection(it)} (${it.toInt()}°)" }
+	private val gforceLat = cdsMetrics.accel.map {
+		it.first?.let {
```

Comment:
I thought the linter doesn't like the shadowed `it`, but perhaps the rules have changed slightly

Reply:
That seems to be no problem. It compiles and runs without errors.
It is also used before. E.g. here:
https://github.com/BimmerGestalt/AAIdrive/blob/a412173f04f5b69f8bf97e4e4d4ea7e4dfac0511/app/src/main/java/me/hufman/androidautoidrive/cds/CDSMetrics.kt#L115-L117

Reply:
Yeah it works, but I remember Android Studio and the build toolchain adding warnings about the variable shadowing.
Not too much of a biggie either way